android/app/src/main/java/com/microsoft/codepush/react/CodePushUpdateUtils.java

```diff
@@ -286,4 +293,4 @@ public static void verifyUpdateSignature(String folderPath, String packageHash,
 
         CodePushUtils.log("The update contents succeeded the code signing check.");
     }
-}
```

Comment:
what this ?

Reply:
If you mean just the last brace then it is mean that you added new line at the end of file. Possible it is so because your automation code-style generator added this new line.

Reply:
thanks, it's right, i found the answer  when open with sublime text2.